### PR TITLE
Migrating layout parser interface to timo

### PR DESCRIPTION
**Pull request by Whattabatt**
This is step 1 to move layout parser from SPP to timo: https://github.com/allenai/scholar/issues/32320

### CODE REVIEW

**Review by yoganandc — APPROVED** [review]
a few notes
- typically we leave a link to the issue this PR is addressing in the description
- shared test fixtures - this means we used the same image for both cases yeah?

@cmwilhelm sorry second time im pinging you but just wanted to say, it feels kinda weird having CI in some other repo.

File: ai2_internal/layout_parser/integration_test.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+from mmda.types.image import tobase64
+
+try:
+    from timo_interface import with_timo_container
+except ImportError as e:
+    logging.warning("""
+    This test can only be run by a TIMO test runner. No tests will run. 
+    You may need to add this file to your project's pytest exclusions.
+    """)
+    sys.exit(0)
```
[the remark this answers is not in the document]
Comment (cmwilhelm) [reply ai2_internal/layout_parser/integration_test.py:37]: No, it's automatically generated by the tool.

**Review by cmwilhelm — APPROVED** [review]
This LGTM.

**Comment by Whattabatt** [comment]
> 

Updated description to link to issue.

The same file page0.png is used to in layoutparser's integration test code in spp and villa's integration test code here

**Pull request merged**